test/Sentry.Tests/PlatformAbstractions/FrameworkInstallationExtensionsTests.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using Sentry.PlatformAbstractions;
+using Xunit;
+
+namespace Sentry.Tests.PlatformAbstractions
+{
+    public class FrameworkInstallationExtensionsTests
+    {
+        [Fact]
+        public void GetVersionNumber_NullFrameworkInstallation_NullVersion()
+        {
+            FrameworkInstallation frameworkInstallation = null;
```

Comment:
This compiles? What happened to nullable reference types?

Reply:
It does, and I added the nullable type in the next commit.